### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by natespilka**
Here are some pictures and an updated README.

### CODE REVIEW

**Comment by natespilka** [comment]
It looks like some unwanted images were copied over. I'll remove them.

File: Individual Assignments/Spilka Nathaniel/week-02/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+I have experience using Stata, Github, Git, R, Matlab, and Bash, but have never used SurveyCTO. I'm looking forward to using more Stata and learning SurveyCTO this semester! In my free time I enjoy cooking, training jiu jitsu, swimming, and playing guitar.
+
+This is what I look like ![priest](/img/pic.JPG)
+
+Here is my favorite place (and favorite people) ![favorite place](/img/pic.HEIC)
+
+I love the MSE!
+
+$$ MSE = \frac{\sum_{i = 1}^{n}(y_{i}-\hat{y}_{i})^{2}}{n} $$
```
Comment (mikegiunti) [inline Individual Assignments/Spilka Nathaniel/week-02/README.md:27]: Everything looks good, but I would revise the math section. I am not super sure how you can do that and I can't preview my changes.

**Review by mikegiunti — CHANGES_REQUESTED** [review]

File: Individual Assignments/Spilka Nathaniel/week-02/README.md (hunk 1 above)
Comment (natespilka) [reply Individual Assignments/Spilka Nathaniel/week-02/README.md:27]: What about this section needs revision? 
Thanks for checking it out

Comment (mikegiunti) [reply Individual Assignments/Spilka Nathaniel/week-02/README.md:27]: When you preview the math, it seems that the Sum symbol is overlapping the i = 1 function. You can preview it by going to changes and pressing the document icon on the top right

**Review by hill-hannah — CHANGES_REQUESTED** [review]
profile picture now shows :) still can't see favorite place picture!

**Comment by natespilka** [comment]
@mikegiunti and @hill-hannah it looks like it's good to go. Hannah stole my favorite place interestingly enough but it's a lovely place so I'll forgive her.

**Review by hill-hannah — APPROVED** [review]

**Review by mikegiunti — APPROVED** [review]

**Comment by natespilka** [comment]
thanks!

**Comment by bbdaniels** [comment]
Great work on this to all of you!

**Pull request merged**